charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.10.7
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 6.7.13
+version: 6.8.0
```

Comment:
Since this PR has backwards compatible, so bumping patch is engough? 🤔

Reply:
If you follow semver specification

```
    MAJOR version when you make incompatible API changes
    MINOR version when you add functionality in a backward compatible manner
    PATCH version when you make backward compatible bug fixes
```

ref : https://semver.org/

Reply:
I was talking about below, the item on PR description. 🙋 

> I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)

Reply:
You speak about this ?

> Minor: New chart functionality (sidecars), major application updates or minor non-backwards compatible changes